.jenkins/cscs-ault/entry.sh

```diff
@@ -38,7 +38,7 @@ sbatch \
     --nodes="1" \
     --partition="${configuration_slurm_partition}" \
     --account="djenkssl" \
-    --time="00:30:00" \
+    --time="02:00:00" \
```

Comment:
Do we need such long time here? As far as I see this is still only building examples and the CUDA/HIP tests, correct?